commands.go

```diff
@@ -343,7 +343,7 @@ func CommandOpen(conf Config, ctx, cmdLine CmdLine) error {
 	for _, task := range ts.Tasks() {
 		urls := xurls.Relaxed.FindAllString(task.Summary+" "+task.Notes, -1)
 		if len(urls) == 0 {
-			return fmt.Errorf("No URLs found in task %v", task.ID)
+			return fmt.Errorf("no URLs found in task %v", task.ID)
```

Comment:
Any reason for un-capitalising? Seems less neat

Reply:
not sure if this is an official best practice or merely a best practice recommended by many code quality tools, but at least the official examples don't capitalize errors , e.g. see https://blog.golang.org/error-handling-and-go

Reply:
Never heard of it but never looked. I'm ok with it though and the docs are good enough of an indication of best practice 👍